lib/views/github-tab-view.js

```diff
@@ -89,4 +93,14 @@ export default class GitHubTabView extends React.Component {
       </div>
     );
   }
+
+  renderHeader() {
+    return (
+      <HeaderView
+        handleProjectSelect={e => this.props.changeProjectWorkingDirectory(e.target.value)}
+        currentProject={this.props.workingDirectory}
+        projectPaths={this.props.project.getPaths()}
```

Comment:
Project paths don't necessarily correspond to git repository working directories. You can have a project path (or multiple project paths!) that are subdirectories of a git working directory, and you can have project paths that have no git working directories.

Finding the candidate git working directories can be fairly expensive - it takes a lot of filesystem accesses to walk up the directory tree. We have a Workdir

We'll need to implement a function (likely way up in `GithubPackage`) that enumerates the available _working directories_ instead, based on the project paths, and then passes them in here to use in the drop-down instead.

Reply:
Noted. I will look into it.

Reply:
(Also, for the record, I'm pretty sure I confused this myself in the feature request writeup. Only realized this while I was doing the review 😬)

Reply:
Believed to be solved